Monika-After-Story/MonikaModDev
# set chess actions dict to defaultdict

closes #9074 

# Key Changes
* changes `_mas_chess_dlg_actions` into a `defaultdict`
* imports `defaultdict` into the global namespace for general use
* adds update script to convert `_mas_chess_dlg_actions`

# Testing
* verify a fresh install has `_mas_chess_dlg_actions` as a `defaultdict`
* Run the update script and verify that `_mas_chess_dlg_actions` gets converted into a `defaultdict`
